### DISCUSSION TITLE
API Integration to produce weekly report extraction

### DISCUSSION THREAD

**Discussion by jeffboon97** [shridhar-tl/jira-assistant#334] [Q&A]
Dear @shridhar-tl , good day to you. may I know if JIRA Assistant support API integration to perform weekly report extraction and sending email?

**Comment by shridhar-tl** [comment]
Hi @jeffboon97 - As of now Jira Assistant does not support automatically sending mails. While this is not feasible with Browser extension and Web version, I will look into the feasibility on Cloud app version of Jira Assistant some time at the beginning of next year.
